examples/files.js

```diff
@@ -117,6 +118,8 @@ var files = {
 		"webgl_loader_utf8",
 		"webgl_loader_vrml",
 		"webgl_loader_vtk",
+		"webgl_loader_wwobj2",
+		"webgl_loader_wwobj2_parallels",
```

Comment:
For clarity... do you mind renaming these to `webgl_loader_obj2_ww` and `webgl_loader_obj2_ww_parallels`?